v3/src/models/tiles/tile-model.ts

```diff
@@ -63,7 +63,9 @@ export const TileModel = types
     // whether to restrict display to certain users
     display: DisplayUserTypeEnum,
     // e.g. "TextContentModel", ...
-    content: TileContentUnion
+    content: TileContentUnion,
+    height: types.maybe(types.number),
+    width: types.maybe(types.number)
```

Comment:
These should not be properties of the tile model -- layout information should only be in the layout. Since the only reference to the new methods appears to be in the nascent mosaic implementation, I think this file can be left unchanged as well.